src/functions/getLatestActiveGame.ts

```diff
@@ -1,17 +1,15 @@
-import { GameData, VoteStates } from '../components/GameGrid/Graph'
 import SinglePoll from '../models/SinglePoll'
-import SinglePollClass from '../models/SinglePoll'
 
-const LatestToOldestStages: (keyof GameData)[] = ['final', 'runnerUp', 'semiFinal', 'quarterFinal', 'groupStages', 'knockout']
+const LatestToOldestStages: (keyof IGameData)[] = ['final', 'runnerUp', 'semiFinal', 'quarterFinal', 'groupStages', 'knockout']
```

Comment:
At some point or other, I've seen a good practice thing where more complex types should be enclosed in `Array<>`
```suggestion
const LatestToOldestStages: Array<keyof IGameData> = ['final', 'runnerUp', 'semiFinal', 'quarterFinal', 'groupStages', 'knockout']
```